src/main/java/step2/domain/Rank.java

```diff
@@ -0,0 +1,34 @@
+package step2.domain;
+
+public enum Rank {
+    FIRST(6, 2000000000),
+    SECOND(5, 1500000),
+    THIRD(4, 50000),
+    FOURTH(3, 5000),
```

Comment:
```suggestion
    FOURTH(3, 5_000),
```
가독성을 위해 요렇게 표기하기도 합니다~